src/test/java/teammates/test/cases/webapi/CreateFeedbackResponseActionTest.java

```diff
@@ -23,13 +50,192 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        useTypicalDataBundle();
+        ______TS("not enough attributes");
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure(Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString());
+        verifyHttpParameterFailure(Const.ParamsNames.FEEDBACK_QUESTION_ID, qn1InSession1InCourse1.getId());
+
+        String[] params = {
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, qn1InSession1InCourse1.getId(),
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+        };
+        ______TS("null recipient Identifier");
+        loginAsStudent(student1InCourse1.getGoogleId());
+        assertThrows(InvalidHttpRequestBodyException.class, () -> {
+            FeedbackResponseCreateRequest createRequest = getResponseRequest(student1InCourse1.getEmail());
+            createRequest.setRecipientIdentifier(null);
+            Action a = getAction(createRequest, params);
+            a.execute();
+        });
+        ______TS("null questionType");
+        assertThrows(InvalidHttpRequestBodyException.class, () -> {
+            FeedbackResponseCreateRequest createRequest = getResponseRequest(student1InCourse1.getEmail());
+            createRequest.setQuestionType(null);
+            Action a = getAction(createRequest, params);
+            a.execute();
+        });
+        ______TS("null responseDetail");
```

Comment:
The three tests above is to test `validate()` method inside `FeedbackResponseCreateRequest`. By right, they should reside in `FeedbackResponseCreateRequestTest`.